lib/nerdtree/path.vim

```diff
@@ -530,8 +530,14 @@ function! s:Path.isUnder(parent)
         return 0
     endif
     for i in range(0, l:that_count-1)
-        if self.pathSegments[i] !=# a:parent.pathSegments[i]
-            return 0
+        if nerdtree#runningWindows()
+            if tolower(self.pathSegments[i]) !=# tolower(a:parent.pathSegments[i])
```

Comment:
Unless I read through the discussion comments too quickly, this still needs addressing. Using `tolower()` should be unnecessary if explicitly using a case-insensitive matching operator.

Reply:
yes, I'm sorry my example was wrong.
something like this should replace the old version.
```vim
if !nerdtree#caseSensitiveFS()
    if self.pathSegments[i] !=# a:parent.pathSegments[i]
```

Reply:
@dangibson
Hello sir,

Do you have any interest in completing this PR? If not I can create a PR containing your commits so I can add the requested changes while keeping the original author of the commits. But to be honest I would love it if you do these changes yourself, Since you already familiarized yourself with the codebase it can be an awesome thing to have somebody else contributing to the project.

If I wanted to implement this by myself it would've been a lot worse, since we had some conversation around implementing it we found many weak points and came up with a more solid implementation as a result. I love working with other developers, All of the good ideas come from people who cooperate for the greater good despite their takes on the subject matter.

I'll delay merging this PR for another week so you have time to read this comment and respond if you want to, And if you don't I totally understand.

Thanks for helping to Improve NERDTree, I've used this plugin since I was a kid in high school so It is literally keeping my childhood alive.

Sincerely yours,
Ali Rezvani